packages/dataviews/src/view-actions.js

```diff
@@ -7,12 +7,13 @@ import {
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { memo } from '@wordpress/element';
+import { settings } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { unlock } from './lock-unlock';
-import { VIEW_LAYOUTS, LAYOUT_TABLE, SORTING_DIRECTIONS } from './constants';
+import { VIEW_LAYOUTS, SORTING_DIRECTIONS } from './constants';
```

Comment:
I checked, and we still need the `icon` declared in `VIEW_LAYOUTS` for the views in the sidebar. We don't need to remove anything more in this PR.  👍

Reply:
> for the views in the sidebar

Yup, that's a detail to explore separately.